src/mds/MDBalancer.cc

```diff
@@ -276,6 +300,17 @@ void MDBalancer::handle_heartbeat(MHeartbeat *m)
     if (mds_load.size() == cluster_size) {
       // let's go!
       //export_empties();  // no!
+
+      int r = mantle_prep_rebalance();
+      if (!r) {
+        mds->clog->info() << "mantle succeeded; "
```

Comment:
Maybe the success message could just go to the MDS log instead of the cluster log?  This will get spammy for anyone who likes to "ceph -w"